roadmap.md

```diff
@@ -22,21 +22,26 @@ We must finish the Taskcluster migration. Supporting both Buildbot and
 Taskcluster within automation is a substantial drag on agility and
 productivity, and is costly in its resource consumption.
 
-### Sunsetting Deprecated Features
+The remaining bits of the migration include:
+ * Porting Windows tests to run natively in Taskcluster
```

Comment:
Maybe this should be:
* Move Windows 7 tests to tier 1 (since they are already ported)